app/models/category.rb

```diff
@@ -1,4 +1,5 @@
 class Category < ApplicationRecord
-  validates :name, presence: true
+  has_many :courses
+  validates :name, presence: true, uniqueness: true, length: { maximum: 100 }
```

Comment:
I think `case_sensitive: false` will be more suitable for category name.

Reply:
Done bhai. Thank you.